contentcuration/contentcuration/frontend/shared/views/LicenseDropdown.vue

```diff
@@ -178,17 +206,30 @@
           ? getLicenseDescriptionValidators().map(translateValidator)
           : [];
       },
+      licences() {
+        return LicensesList.filter(license => license.id).map(license => ({
+          ...license,
+          name: this.translateConstant(license.license_name),
+          description: this.translateConstant(license.license_name + '_description'),
+        }));
+      },
     },
     methods: {
-      ...mapMutations({
-        setShowAboutLicenses: 'SET_SHOW_ABOUT_LICENSES',
-      }),
       translate(item) {
         if (item.id === MIXED_VALUE) {
           return this.$tr('mixed');
         }
         return (item.id && item.id !== '' && this.translateConstant(item.license_name)) || '';
       },
+      toggleAboutLicenseDisplay() {
+        this.showAboutLicense = !this.showAboutLicense;
+      },
+      getLicenseUrl(license) {
+        const url = new URL(license.license_url);
```

Comment:
Hi @LianaHarris360  is the `license.license_url` a guaranteed non null, non empty, url string? If not it might he worthwhile adding a `try...catch` to prevent any failures. 

Otherwise the rest of change seems logical to me and fixes the issue with the codeQL. We should be good to merge this back to unstable once once the above question is clarified. 

Thanks

Reply:
It would be good to include a `try...catch` here, thanks for pointing this out! All entries, with the exception of [one](https://github.com/learningequality/studio/blob/f029bd49345be27b526ee79ecc3565ceb8d1866a/contentcuration/contentcuration/frontend/shared/leUtils/Licenses.js#L105), are guaranteed to be non-empty URL strings.